CIP-1694/README.md

```diff
@@ -0,0 +1,624 @@
+  The type of governance action. Note that the three possible treasury actions involve Lovelace amounts $T_0$, $T_1$, $T_2$, and $T_3$.
+
+* **Constitutional Committee**<br/>
+  A value of :heavy_check_mark: indicates that `Quorum-Many` Constitutional Committee 'yes' votes are required.<br/>
+  A value of :x: means that Constitutional Committee votes do not apply.
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*, ranging from 0 to 100 (inclusive).
+
+* **AVST**<br/>
+  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*.
+
+* **AVST action**<br/>
+  The consequence of the voting stake being below the AVST.
+  * **Required**<br/>
+    This governance action cannot be ratified without achieving the AVST threshold.
+  * **SPO**<br/>
+    For this governance action, SPO vote endorsement is not required if there is a sufficient active voting stake.
+    Otherwise, SPO vote endorsement is required.
+
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
+
+| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | AVST Action | SPOs     |
+| ---                                                  | :---:                    | ---      | ---      | ---         | ---      |
+| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | Required    | $R_1$    |
+| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO         | $R_{2a}$ |
+| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | Required    | $R_{2b}$ |
+| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO         | $R_3$    |
+| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -           | $R_4$    |
+| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | SPO         | $R_5$    |
+| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | SPO         | $R_{6a}$ |
+| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | SPO         | $R_{6b}$ |
+| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | SPO         | $R_{6c}$ |
+
+Some of the parameters given in this table ( $P_1$ ... $R_{6c}$, $T_1$ ... $T_3$ ) may be updatable protocol parameters, but others should be hard-coded. This proposal deliberately leaves both this choice and the choice of actual parameter values open for discussion.
```

Comment:
You can leave some stuff not coded because you want to give people the right to make an appeal if something for example went against the constitution and it was up to a governing body to decide the final outcome of that appeal.

Reply:
Thanks @Kronoshus . Re-reading the text, it does state that it is purposly left out!

Reply:
> > Applying this lesson to social decentralization and governance it would be wise to limit the voting power of all SPO's to some leverage limit multiple of the SPO's pledge: L. This has a profound effect: there is an added social/governance motivation for pledge. Groups like Binance and Coinbase which do not pledge any ADA would have no voting power.
> 
> I think this is a very valid point, thank you @michael-liesenfelt . How do we reconcile this, however, with the fact that SPOs that do not wish for a hard fork can always just not upgrade their software? If they do that, as a collective, then the 'no' vote is basically determined by the longest chain, which ignores pledge.

Need to allow for the continual fragmentation of groups. Within SPO's you need to have a mechanism that allows them to split from the main vote and give their vote elsewhere. This is assuming you have a yes or no proposal system rather than one that allows fragmentation of yes or no options.